.circleci/config.yml

```diff
@@ -8,7 +8,7 @@ jobs:
     build:
         machine: true
         environment:
-            IMAGE_NAME: stellargroup/phylanx:dev
+            IMAGE_NAME: stellargroup/phylanx
```

Comment:
What's the rationale to change this name?

Reply:
I separated the prerequisites image and the current working master.